scss/widgets/base/scheduler/_tooltip.scss

```diff
@@ -0,0 +1,14 @@
+$SCHEDULER_APPOINTMENT_BASE_COLOR: null !default;
```

Comment:
It is not a good idea to have both scheduler/ (folder) and _scheduler.scss file in the `base` folder. It can cause a collision while import `base/scheduler` module. (The same in `common` folder.)